chrome/manifest.json

```diff
@@ -1,16 +1,22 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
 
   "name": "BRemover",
   "description": "BRemover can modify how certain webs work given the rules by the user.",
   "version": "1.0",
 
-  "browser_action": {
-    "default_icon": "icon.png",
-    "default_popup": "popup.html"
+  "icons": {
+    "128": "icon.png"
   },
   "background": {
-    "scripts": ["background.js"]
+    "service_worker": "background.js"
   },
-  "permissions": ["tabs", "webNavigation", "*://*/*", "storage", "downloads"]
+  "action": {
+    "default_popup": "popup.html",
+    "default_icon": {
+      "128": "icon.png"
+    }
+  },
+  "permissions": ["tabs", "webNavigation", "storage", "downloads", "scripting"],
+  "host_permissions": ["<all_urls>"]

```

Comment:
If we prefer instead of using `host_permissions: [all_urls]` we can add one more permission called `activeTab`. It seems like this one is better as there is no warning message on installation.

Reply:
I tried using the `activeTab` permission instead of `host_permissions: [<all_urls>]`, but it doesn't work like the documentation says.